addons/auth_password_policy/static/src/password_meter.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import {sprintf} from "@web/core/utils/strings";
+import {computeScore, Policy} from "./password_policy";
+
+const { Component, xml } = owl;
+
+export class Meter extends Component {
+    get title() {
+        return sprintf(
+            this.env._t("Required: %s\n\nHint: to increase password strength, increase length, use multiple words, and use non-letter characters."),
+            String(this.props.required) || this.env._t("no requirements"),
+        );
+    }
+
+    get value() {
+        return computeScore(
+            this.props.password,
+            this.props.required,
+            this.props.recommended,
+        )
```

Comment:
code should be linted/prettified (you can use the web tooling, you just need to whitelist this addon)

Reply:
I have no idea what that means.

Reply:
The tooling can be enabled with `./addons/web/tooling/enable.sh`. It automatically lints/prettifies the modified files when committing. However, the script applies on a whitelist of folders/files (see https://github.com/odoo/odoo/blob/master/addons/web/tooling/_eslintignore and https://github.com/odoo/odoo/blob/master/addons/web/tooling/_prettierignore [1]).

[1] yes, we should remove one of them